src/Vegetation/PlantHydraulics.jl

```diff
@@ -113,18 +113,22 @@ $(DocStringExtensions.FIELDS)
 struct PlantHydraulicsParameters{FT <: AbstractFloat, PSE}
     "RAI, SAI, LAI in [m2 m-2]"
     area_index::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
-    "water conductivity in the different plant compartments (m/s) when pressure is zero, a maximum"
-    K_sat::NamedTuple{(:root, :stem, :leaf), Tuple{FT, FT, FT}}
-    "van Genuchten parameter"
-    vg_α::FT
-    "van Genuchten parameter"
-    vg_n::FT
-    "van Genuchten parameter"
-    vg_m::FT
-    "porosity"
-    ν::FT
-    "storativity"
-    S_s::FT
+    "water conductivity in the above-ground plant compartments (m/s) when pressure is zero, a maximum"
+    above_ground_plant_K_sat::NamedTuple{(:stem, :leaf), Tuple{FT, FT}}
+    "water conductivity in the roots (m/s) when pressure is zero, a maximum"
+    K_sat_root::Function
```

Comment:
Why add this?

Reply:
I thought I had too as K_sat_root was of a different type (function) than the above_ground_plant_K_sat values (FT), but maybe not necessary and can include a function in the Tupple? Somthing like Tupple{FT,FT,Function} ?

Reply:
fixed this